internal/common/rules/ruleerror.go

```diff
@@ -0,0 +1,67 @@
+package rules
+
+import (
+	"fmt"
+
+	"github.com/SOMAS2020/SOMAS2020/pkg/miscutils"
+)
+
+// RuleErrorType is a non-critical issue which can be caused by an island trying to modify, register or pick rules which isn't mechanically feasible
+type RuleErrorType int
+
+const (
```

Comment:
can you add a block level comment here to shut linter up? optional...

Reply:
Agreed and changed